Package.swift

```diff
@@ -163,12 +159,7 @@ let package = Package(
             dependencies: [
                 .target(name: "Apodini"),
                 .target(name: "ApodiniVaporSupport"),
-                .product(name: "Fluent", package: "fluent"),
-                .product(name: "FluentKit", package: "fluent-kit"),
-                .product(name: "FluentMongoDriver", package: "fluent-mongo-driver"),
-                .product(name: "FluentSQLiteDriver", package: "fluent-sqlite-driver"),
```

Comment:
Removing the SQLLite drivers should allow us to remove the sqlite dependency installs currently happening in the CI right?

Reply:
Sadly no. We still use the in memory database SQLite driver for the unit tests and it therefore it is still required to have the dependencies installed in the CI environment.

Reply:
Maybe related to some missing understanding of fluent internals, but we can use the sqlite driver without specifying it as a dependency in our `Package.swift`?

Reply:
It is still defined as a package dependency (line 46 in Package.swift) but just not a dependency in the ApodiniDatabase target and only in the ApodiniTests target. The dependency is removed here but we still use it in the other test target.

Reply:
Ah, I completely confused the dependencies section 😅 Thanks for the clarification.